BackendServer/Models/Message.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BackendServer.Models
+{
+    internal abstract class Message(string content)
+    {
+        private string content = content;
+        public string Content
+        {
+            get { return content; }
+            set { content = value; }
+        }
+
+        private DateTime timestamp;
```

Comment:
Where will timestamp be initialized?

Reply:
Timestamps need to persist because chat history persists, think discord chat. Perhaps the protocol should include timestamps as you said.

Reply:
Updated protocol to include timestamps